package.json

```diff
@@ -118,18 +118,18 @@
     "lint-fix": "eslint --fix --cache ./",
     "build": "babel src/ -d lib/ --copy-files",
     "watch": "babel --watch src/ -d lib/ --copy-files",
-    "test:mongodb:runnerstart": "cross-env MONGODB_VERSION=${MONGODB_VERSION:=$npm_config_dbversion} MONGODB_TOPOLOGY=${MONGODB_TOPOLOGY:=standalone} mongodb-runner start",
+    "test:mongodb:runnerstart": "cross-env MONGODB_VERSION=${MONGODB_VERSION:=$npm_config_dbversion} MONGODB_TOPOLOGY=${MONGODB_TOPOLOGY:=standalone} mongodb-runner start -t standalone --version $npm_config_dbversion -- --port 27017",
```

Comment:
We are getting the topology via `MONGODB_TOPOLOGY=${MONGODB_TOPOLOGY:=standalone} `; I believe this should be passed to the runner instead of a fixed `standalone`?

Reply:
Why was this resolved?